### PR TITLE
Less cold cold

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## Description
ACHOOO
I think someone got a cold...

## Motivation and Context
Well thats rather to cold for people to handle

## How Has This Been Tested?
Its not

## Screenshots (if appropriate):
N/A
## Changelog (necessary)
:cl:
tweak: Snow from 180kevinz to 248kevinz
/:cl:

### CODE REVIEW

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 1]
```diff
@@ -93,7 +93,7 @@
 	ore_type = /obj/item/stack/sheet/mineral/snow
 	planetary_atmos = TRUE
 	floor_tile = null
-	initial_gas_mix = "o2=22;n2=82;TEMP=180"
+	initial_gas_mix = "o2=22;n2=82;TEMP=248"
```
Comment (Laus4Deus) [inline code/game/turfs/simulated/floor/fancy_floor.dm:96]: I can see so many things being fixed by this... 😞 
Thanks, tribadism space clone

**Comment by Goldrobo** [comment]
So... do this just make the cold weather less cold, or...?

**Comment by Trilbyspaceclone** [comment]
@Goldrobo You know the snow tile?
It makes those not be as cold.
Not the weather.

**Comment by Trilbyspaceclone** [comment]
Code wise its based on tile/turf. Not wealther by the by. IDK if you all use snow storm but iirc they deal -231c were im used to codeing through suits that are not ash proof*

**Pull request merged**